codility_training/lessons.lesson09.MaximumSliceProblem.MaxProfit/khw970421.js

```diff
@@ -0,0 +1,18 @@
+//정확도 100 효율성 75
+
+function solution(A) {
+    let max=0;
+    let count=0;
+    let p=0;
+    let A_size=A.length;
```

Comment:
```suggestion
    const A_size=A.length;
```
size는 변하지 않으므로 상수로 선언

Reply:
이 부분은 효율성맞추려고 정신없어서 원래 하던건데도 잊어먹었네요